packages/lending/src/protocols/aave-v2/lending-protocol.ts

```diff
@@ -345,51 +345,56 @@ export class LendingProtocol extends Protocol {
       if (token.isWrapped) continue;
 
       const reserveData = reserveDataMap[token.address];
-      const assetPrice = assetPriceMap[token.address];
-      const userBalance = userBalancesMap[token.address];
+      if (!reserveData) continue;
 
-      let usageAsCollateralEnabled = reserveData.usageAsCollateralEnabled;
-      if (Number(userBalance.supplyBalance) > 0) {
-        usageAsCollateralEnabled = usageAsCollateralEnabled && userBalance.usageAsCollateralEnabled;
-      }
+      const {
+        usageAsCollateralEnabled: _usageAsCollateralEnabled,
```

Comment:
follow the naming convention of Ln360, should _usageAsCollateralEnabled be `reserveDataUsageAsCollateralEnabled`?

Reply:
used `_usageAsCollateralEnabled` to keep it shorter,  `reserveDataUsageAsCollateralEnabled` works for me too, lets keep the same naming convention.

Reply:
updated